qa/pull-tester/rpc-tests.py

```diff
@@ -203,6 +203,7 @@ def option_passed(option_without_dashes):
     'abandonconflict',
     'p2p-versionbits-warning',
     'importprunedfunds',
+    'thinblocks'
```

Comment:
Principal ACK on name, but I'm not really aware of thinblocks functionality being exposed apart from Xthin.
Other reviewers might correct me here.

I would have chosen xthinblocks.py but the name is fine.